sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -2606,6 +2606,13 @@ class DataSourceV2SQLSuite
     }
   }
 
+  private def testNotSupportedV2Command(sqlCommand: String, sqlParams: String): Unit = {
+    val e = intercept[AnalysisException] {
+      sql(s"$sqlCommand $sqlParams")
+    }
+    assert(e.message.contains(s"$sqlCommand is not supported for v2 tables"))
```

Comment:
Not related to this PR, but we should unify the error message, to either  `is not supported for v2 tables` or `is only supported with v1 tables`

Reply:
Yes, the plan is to move all the commands using `parseV1Table` in `ResolveSessionCatalog` to use the new framework and unify the message in the process.